build.sbt

```diff
@@ -171,6 +171,7 @@ Global / onLoad ~= { old =>
   old
 }
 Global / cancelable := true
+Global / excludeLintKeys += scalafixConfig
```

Comment:
What does this do?

Reply:
sbt wrongly warns that `scalafixConfig` setting isn't used

Reply:
I just opened https://github.com/scalacenter/sbt-scalafix/pull/233 to avoid that